assets/css/components/text-grid.scss

```diff
@@ -16,6 +16,8 @@
         &__figure {
             /*figure tag*/
            @apply pb-30 xl:w-1/5 pr-20;
+           @apply w-230 h-130 flex justify-center items-center;
+           @apply md:w-210; 
```

Comment:
![image](https://user-images.githubusercontent.com/16774319/142203915-8e1f8079-5848-46d9-85e9-a4308d957bc8.png) de logo's zijn niet in het midden. dit komt waarschijnlijk door de pr-20; zorg ervoor dat de padding alleen op grote schermen worden toegepast